analysis_driver/dataset.py

```diff
@@ -344,6 +346,31 @@ def _find_pooling_step_for_artifact(art, expected_pooling_step_name=None, max_it
             )
         return art.input_artifact_list()
 
+    @property
+    def rapid_samples_by_lane(self):
+        """
+        Search the sequencing run in the Lims for any samples with the UDF 'Rapid Analysis' set, and return their sample
+        ID, project ID and all UDFs.
+        """
+        if self._rapid_samples_by_lane is None:
+            self._rapid_samples_by_lane = {}
+
+            # TODO: move away from using the Lims API
```

Comment:
I agree with Luke's original comment that `TODO` by themselves do not allow us to track the tasks. 
I've open an issue to limit the use of the LIMS API, can you reference #394 in the `TODO`